live-examples/css-examples/lists/list-style-position.html

```diff
@@ -21,11 +21,11 @@
         <div>
             <p>NASA Notable Missions</p>
             <ul id="example-element" class="transition-all">
-                <li>Apollo</li>
-                <li>Hubble</li>
-                <li>Chandra</li>
-                <li>Cassini-Huygens</li>
-                <li>Spitzer</li>
+                <li>Apollo 11: First Moon Men</li>
+                <li>City in Space: The International Space Station</li>
+                <li>The Columbia Tragedy: Feb. 1, 2003</li>
+                <li>Great Observatory: The Hubble Space Telescope</li>
+                <li>Jan. 28, 1986: Challenger Accident</li>
```

Comment:
Maybe it's just me, but could we have fewer tragedies? Maybe:

```suggestion
                <li>Great Observatory: The Hubble Space Telescope</li>
                <li>Everlasting Mars Rovers</li>
```